apps/dashboard/app/models/user_configuration.rb

```diff
@@ -66,11 +66,13 @@ class UserConfiguration
     ConfigurationProperty.property(name: :interactive_apps_menu, default_value: []),
 
     # Custom pages configuration property
-    ConfigurationProperty.property(name: :custom_pages, default_value: {})
+    ConfigurationProperty.property(name: :custom_pages, default_value: {}),
+    # Support ticket configuration property
+    ConfigurationProperty.property(name: :support_ticket, default_value: {})
   ].freeze
 
-  def initialize(request_hostname: nil)
-    @config = ::Configuration.config
+  def initialize(config: nil, request_hostname: nil)
+    @config = config || ::Configuration.config
```

Comment:
We're changing this interface just for the test cases? I'm not so sure about that...

Reply:
Yes - It was to make it easier to test.
Will revert it back.

Reply:
fixed